pkgs/development/python-modules/ludios_wpull/default.nix

```
@@ -15,15 +15,15 @@
 
 buildPythonPackage rec {
   pname = "ludios_wpull";
-  version = "3.0.7";
+  version = "3.0.9";
 
   disabled = (!isPy3k) || (pythonAtLeast "3.8");
```

```suggestion
  disabled = pythonAtLeast "3.8";
```
The project does not support python2 anymore https://github.com/ArchiveTeam/ludios_wpull/blob/master/setup.py#L81

---

Thanks. I removed the `isPy3k` and bumped the disabled version to 3.9+, where ludios_wpull is incompatible.